clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/description/manifest/KubernetesKind.java

```diff
@@ -54,6 +54,10 @@
       createWithAlias("daemonSet", "ds", KubernetesApiGroup.APPS);
   public static final KubernetesKind DEPLOYMENT =
       createWithAlias("deployment", "deploy", KubernetesApiGroup.APPS);
+  public static final KubernetesKind CSI_DRIVERS =
+      createWithAlias("csidriver", null, KubernetesApiGroup.STORAGE_K8S_IO);
+  public static final KubernetesKind CSI_NODES =
+      createWithAlias("csinode", null, KubernetesApiGroup.STORAGE_K8S_IO);
```

Comment:
Shouldn't it be `csiNode` and `csiDriver` (camel case)?
Also very minor comment: sort alphabetically, so after cronJob.